.github/workflows/continuous-integration.yml

```diff
@@ -91,6 +91,71 @@ jobs:
       - name: Annotate Sass lint results
         run: node ./script/github-actions/sasslint-annotation-format.js
 
+  contract-tests:
+    name: Contract Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"
+
+      - name: Get bot token from Parameter Store
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /dsva-vagov/pact-broker/utility/PACT_BROKER_BASIC_AUTH_PASSWORD
+          env_variable_name: PACT_BROKER_BASIC_AUTH_PASSWORD
+
+      - name: Get Node version
+        id: get-node-version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.get-node-version.outputs.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --prefer-offline
```

Comment:
Do you have a link to any docs for `--prefer-offline`? This is the best I could find:
>`'--prefer-offline'`, 'use network only if dependencies are not available in local cache'

Source: https://github.com/yarnpkg/yarn/commit/d5a6f222e7ab7e25b9d8e1a12a158c24003ac677

Reply:
It's not very well documented. I've only seen it as one of the CLI options from `yarn --help`.